uber/templates/regform.html

```diff
@@ -1,3 +1,5 @@
+{% import 'macros.html' as macros %}
```

Comment:
Theoretically this file (regform.html) is included in files that extend base.html, so macros.html should be imported there. Did it not work despite that?